sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cloudfoundry/AzureCfService.java

```diff
@@ -26,7 +26,7 @@ enum AzureCfService {
             "accessKey", "account-key"
         )),
 
-    REDIS("spring.redis", "azure-rediscache",
+    REDIS("spring.data.redis", "azure-rediscache",
```

Comment:
why change this?

Reply:
New prefix, https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java#L35

Reply:
Yes, https://github.com/Azure/azure-sdk-for-java/blob/1c3ba25b93231fc126327778d25cdcfde52461af/sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/cloudfoundry/AzureCloudFoundryEnvironmentPostProcessorTests.java#L54